aspnetcore/security/authorization/secure-data/samples/final15/Authorization/ContactRoleAuthorizationHandler.cs

```diff
@@ -16,7 +16,7 @@ public class ContactRoleAuthorizationHandler
                 return Task.FromResult(0);
             }
 
-            if (context.User.IsInRole(requirement.Name))
+            if (context.User.IsInRole(Constants.OperationRolePrefix + requirement.Name))
```

Comment:
@Rick-Anderson I understood guest as any authenticated user that is not in any role, ie any user registered via the ui. I was going to allow guest to view the data, but not allow anonymous. So the only users who will be able to edit at all are the seeded users. Does that make sense?